reactive/kubernetes_worker.py

```diff
@@ -1147,16 +1147,16 @@ def request_integration():
         cloud = endpoint_from_flag('endpoint.gcp.joined')
         cloud.label_instance({
             'k8s-io-cluster-name': cluster_tag,
-            'juju.io/cloud': 'gce',
-            'juju.io/az': os.environ.get('JUJU_AVAILABILITY_ZONE', ''),
+            'juju-io-cloud': 'gce',
+            'juju-io-az': os.environ.get('JUJU_AVAILABILITY_ZONE', ''),
         })
         cloud.enable_object_storage_management()
     elif is_state('endpoint.azure.joined'):
```

Comment:
hi @johnsca 
Is it possible to add 
```python
elif is_state('endpoint.openstack.joined'):
    # tag node with `'juju-io-cloud': 'openstack',`
    ...
```
here?

Reply:
Please see https://github.com/charmed-kubernetes/charm-kubernetes-worker/pull/7 instead.  The node tags aren't actually useful and these are being removed entirely in favor of annotations on the SC as discussed with @wallyworld.